db/seeds.rb

```diff
@@ -34,12 +34,291 @@
+      <Suffix xsi:nil="true" />
+    </OwnerPoOvm>
+  </CertificateOwner>
+  <ContactPerson>
+    <ContactName>{{ Kontaktná osoba*:text_field }}</ContactName>
+    <Email>{{ Email*:email_field }}</Email>
+    <Phone>{{ Telefón*:text_field }}</Phone>
+  </ContactPerson>
+  <AuthentificationCertificateData>
+    <Thumbprint>{{ Digitálny odtlačok*:text_field }}</Thumbprint>
+  </AuthentificationCertificateData>
+  <TechnicalAccountData>
+    <TechnicalAccountName>{{ Názov technického účtu*:text_field }}</TechnicalAccountName>
+  </TechnicalAccountData>
+</AuthentificationCertificateRegistration>',
+  metadata: {
+    sktalk_class: 'IAM_FORM_REGISTRACIA_AC_REQ_01',
+    posp_id: '42156424.IAM_Zmena_zapisu_autentifikacneho_certifikatu.sk',
+    posp_version: '1.5',
+    message_type: '42156424.IAM_Zmena_zapisu_autentifikacneho_certifikatu.sk'
+  },
+  system: false
+)
+Upvs::MessageTemplate.find_or_create_by!(
+  name: 'IRVIN - Žiadosť o zrušenie prístupových oprávnení technického účtu (zrušenie technického účtu)',
+  content: '<TechnicalAccountTermination
+			xmlns="http://schemas.gov.sk/form/42156424.IAM_Zrusenie_TU.sk/1.5"
+			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+			<CertificateOwner>
+				<IdentityType>PO</IdentityType>
+				<OwnerPoOvm>
+					<ICO>{{ IČO*:text_field }}</ICO>
+					<Suffix xsi:nil="true"/>
+				</OwnerPoOvm>
+			</CertificateOwner>
+			<ContactPerson>
+        <ContactName>{{ Kontaktná osoba*:text_field:"Ján Suchal" }}</ContactName>
+        <Email>{{ Email*:email_field:"jan.suchal@test.sk" }}</Email>
+        <Phone>{{ Telefón*:text_field:"+4190000000" }}</Phone>
+			</ContactPerson>
+			<TechnicalAccountData>
+				<DateFrom>{{ Zrušenie technického účtu k dátumu a času*:date_field }}</DateFrom>
+        <TechnicalAccountName>{{ Názov technického účtu*:text_field }}</TechnicalAccountName>
+			</TechnicalAccountData>
+		</TechnicalAccountTermination>',
+  metadata: {
+    sktalk_class: 'IAM_FORM_ZNEPLATNENIE_TU_REQ_01',
+    posp_id: '42156424.IAM_Zrusenie_TU.sk',
+    posp_version: '1.5',
+    message_type: '42156424.IAM_Zrusenie_TU.sk'
+  },
+  tenant: tenant,
+  system: false
+)
+Upvs::MessageTemplate.find_or_create_by!(
+  name: 'Žiadosť o zrušenie prístupových oprávnení technického účtu (zrušenie technického účtu)',
+  content: '<TechnicalAccountTermination
+			xmlns="http://schemas.gov.sk/form/42156424.IAM_Zrusenie_TU.sk/1.5"
+			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+			<CertificateOwner>
+				<IdentityType>PO</IdentityType>
+				<OwnerPoOvm>
+					<ICO>{{ IČO*:text_field }}</ICO>
+					<Suffix xsi:nil="true"/>
+				</OwnerPoOvm>
+			</CertificateOwner>
+			<ContactPerson>
+        <ContactName>{{ Kontaktná osoba*:text_field }}</ContactName>
+        <Email>{{ Email*:email_field }}</Email>
+        <Phone>{{ Telefón*:text_field }}</Phone>
+			</ContactPerson>
+			<TechnicalAccountData>
+				<DateFrom>{{ Zrušenie technického účtu k dátumu a času*:date_field }}</DateFrom>
```

Comment:
Tu mame trochu problem. Bolo by potrebne pouzit datetime picker (co neviem ci chceme robit a ci vieme jednoducho). Alebo robit nejaku vyhybku. Na slovensko.sk to je takto. 
![image](https://github.com/slovensko-digital/govbox-pro/assets/43437253/f6785d09-9989-440e-bbfd-3e28006ab254)
Co navrhujes @jsuchal?

Reply:
Taketo nevieme? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local

Reply:
To uz vyzera, ze potrebujeme do template jazyka aj nejake transformacie, ale pockal by som kym budeme mat dalsi taky pripad. Nevieme tutu len do sablony dat navrdo `:00` ? A nemusime hackovat?

Reply:
Upravene.